slides/src/content/introduction.md

```diff
@@ -37,4 +53,247 @@ FIXME
+- `src` folder -- application code
+- `tsconfig.json` -- TypeScript configuration
+
+---
+
+## TypeScript
+
+- TypeScript is exactly like JavaScript, but with types
+- Has ES6 and ES7 features
+- Compiles to JavaScript before being run in the browser
+- No runtime support for types
+
+---
+
+## First Look at the Component
+
+```typescript
+import { Component } from '@angular/core';
+
+@Component({
+  selector: 'app-root',
+  template: '<h1>{{title}}</h1>',
+  styles: ['h1 { color: font-size: 30px; }']
+})
+export class AppComponent {
+  title = 'app works!';
+}
+```
+
+- Import statement gives you access to Angular components
+- `@Component` is a decorator function that tells Angular this class is a component
+- `selector` is the way you intend to reference the component. Used like an HTML tag `<app-root></app-root>`
+- `template` is what gets rendered onto the page. For web applications, this is HTML
+- `styles` are css stylesheets that get applied to your component
+- `AppComponent` is a class that decides how the component should act
+
+---
+
+## What is a Component?
+
+- The core building block of an Angular application is the component
+- In Angular, a component is some chunk of application logic combined with something that gets displayed
+- Everything being displayed on any page should be associated with some component
+
+---
+
+## Component Template Syntax
+
+- `<h1>{{title}}</h1>` represents __interpolation__
+- Interpolation attempts to evaluate the expression between the curly braces and turn that into a string
+- Symbols (such as title) are evaluated within an expression context, in this case, the component's class members
+
+---
+
+## Generating a New Component
+
+- We'll use Angular CLI to create a new component
+- `ng generate component [component path]`
+- Component gets added to the list of declarations
+- Generated component follows the same structure as `AppComponent`
+- Prefix `app` on both components. This is an Angular best practice
+
+---
+
+## Integrating a Component
+
+Every component has a selector:
+
+*example.component.ts*
+```typescript
+@Component({
+	selector: 'app-example',
+	template: '<p>Multiple component example</p>'
+})
+export class ExampleComponent { }
+```
+
+---
+
+## Integrating a Component
+
+Use selector as an HTML tag within another component's template
+
+
+*app.component.ts*
+```typescript
+@Component({
+	selector: 'app-root',
+	template: `
+	  <h1>My Angular App</h1>
+	  <app-example></app-example>
+	`
+})
+export class AppComponent { }
+```
+[View Example](https://plnkr.co/edit/CMGx2TLUdPcL0D4tvGI9?p=preview)
+
+
+---
+
+## Conditional Rendering
+
+- Done using an Angular __directive__
+- __Directives__ are some logic that affects how some chunk of the page is rendered
+- To render conditionally, Angular provides the built in `ngIf` directive
+
+---
+
+## Directive Syntax
+
+- Used within a component's template
+- Invoked by using a data binding
```

Comment:
@gvwilson I actually think this slide is invalid

The second (and subsequent third/fourth) points are not actually necessary parts of a directive

A directive is just like a component, the only difference is, it has no view (template).
A directive selector matches are the same as a component's (they can be elements, attributes, etc.). Most commonly it's an attribute selector.

We don't actually need to introduce square brackets or data binding here at all

This may actually be better as we can at a high level explain why `ngIf` requires an asterisk

This slide may be better if it were converted from "Directive Syntax" (because there really isn't such a thing), to "What is a Directive"